src/device-registry/.env.development.template

```
@@ -199,4 +199,7 @@ DEV_ADMIN_SETUP_SECRET=
 DEV_HCAPTCHA_SECRET_KEY=
 
 # Environment Defaults
-DEV_DEFAULT_COHORT=
\ No newline at end of file
+DEV_DEFAULT_COHORT=
+# Comma-separated cohort names that cannot be assigned to users/groups (e.g. "airqo,another_system_cohort").
+# "airqo" is always blocked by default even without this variable.
+DEV_PROTECTED_COHORT_NAMES=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)